core/src/main/java/tc/oc/pgm/menu/InventoryMenuUtils.java

```diff
@@ -0,0 +1,160 @@
+package tc.oc.pgm.menu;
+
+import java.util.ArrayList;
+import java.util.List;
+import net.kyori.text.TranslatableComponent;
+import tc.oc.pgm.api.match.Match;
+
+/** A collection of some static methods for working with building {@link InventoryMenu}s */
+// TODO: Abstract out adding a row for pagination?
+public class InventoryMenuUtils {
+
+  /**
+   * A {@link InventoryMenu} that fits any amount of {@link InventoryMenuItem}s
+   *
+   * @param match the match the inventory will exist in
+   * @param title the title of the inventory
+   * @param itemsWithoutSpaces a list of all {@link InventoryMenuItem}s that should be displayed on
+   *     this menu
+   * @return a pretty menu that fits any amount of {@link InventoryMenuItem}s
+   */
+  public static InventoryMenu prettyMenu1(
```

Comment:
Maybe instead of prettyMenu1 2 3 etc, you could make it more informative. For example

1-> `prettyMenu()`

2-> `smallMenu()`

3 -> `normalMenu()`

Feel free to use other names, but just so you get the general idea 😄